doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -0,0 +1,599 @@
+
+Update Grub for the Ubuntu Service VM
+=====================================
+
+#. Update the ``/etc/grub.d/40_custom`` file as shown below. 
+
+   .. note::
+      Enter the command line for the kernel in ``/etc/grub.d/40_custom`` as
+      a single line and not as multiple lines. Otherwise, the kernel will
+      fail to boot.
+
+   **menuentry 'ACRN Multiboot Ubuntu SOS' --id ubuntu-service-os**
+
+   .. code-block:: bash
+
+      {
+
+        load_video
+        insmod gzio
+        insmod part_gpt
+        insmod ext2
+
+        search --no-floppy --fs-uuid --set 9bd58889-add7-410c-bdb7-1fbc2af9b0e1
+        echo 'loading ACRN...'
+        multiboot2 /boot/acrn.bin  root=PARTUUID="e515916d-aac4-4439-aaa0-33231a9f4d83"
+        module2 /boot/bzImage Linux_bzImage
+
+      }
+
+   .. note::
+
+      Adjust this to your uuid and PARTUUID for the root= parameter with
+      lbkid cmdline (or use the device node directly).
+
+      Update the kernel name if you used a different name as the source
+      for your Service VM kernel.
+
+#. Modify the ``/etc/default/grub`` file to make the grub menu visible when
+   booting and make it load the Service VM kernel by default. Modify the
+   lines shown below:
+
+   .. code-block:: none
+
+      GRUB_DEFAULT=ubuntu-service-os
+      #GRUB_TIMEOUT_STYLE=hidden
+      GRUB_TIMEOUT=5
+
+#. Update Grub on your system:
+
+   sudo update-grub
+
+Reboot the system
+=================
+
+Reboot the system. You should see the Grub menu with the new **ACRN ubuntu-service-os** entry. Select it and proceed to booting the platform. The system will start Ubuntu and you can now log in (as before).
+
+To verify that the hypervisor is effectively running, check ``dmesg``. The typical output of a successful installation resembles the following:
+
+.. code-block:: none
+
+   dmesg | grep ACRN
+   [    0.000000] Hypervisor detected: ACRN
+   [    0.862942] ACRN HVLog: acrn_hvlog_init
+
+
+Additional settings in the Service VM
+=====================================
+
+BIOS settings of GVT-D for WaaG
+-------------------------------
+
+.. note::
+   Skip this step if you are using a KBLNUC.
+
+Go to **Chipset** -> **System Agent (SA) Configuration** -> **Graphics
+Configuration** and make the following settings:
+
+The **DVMT Pre-Allocated** to **64MB**:
+
+.. figure:: images/DVMT-reallocated-64mb.png
+
+Set **PM Support** to **Enabled**: 
+
+.. figure:: images/PM-support-enabled.png
+
+OVMF for User VM launching
+--------------------------
+
+The User VM will be launched by the OVMF, so copy it to the specific folder:
+
+.. code-block:: none
+
+   $ sudo mkdir -p /usr/share/acrn/bios
+   $ sudo cp /home/acrn/work/acrn-hypervisor/devicemodel/bios/OVMF.fd  /usr/share/acrn/bios
+
+Install IASL in Ubuntu for User VM launch
+-----------------------------------------
+
+ACRN uses ``iasl`` to parse **User VM ACPI** information. The original ``iasl``
```

Comment:
@fuzhongl, I updated this paragraph for grammatical clarity. Please re-read to make sure I did not alter the intended message.

Reply:
confirmed.